pandas/tests/io/json/test_normalize.py

```diff
@@ -375,3 +375,27 @@ def test_nonetype_dropping(self):
              'info.last_updated': '26/05/2012'}]
 
         assert result == expected
+
+    def test_nonetype_inner_most_level(self):
```

Comment:
Can you check if we have a test case to handle None values that show up in the initial level?

Reply:
Thanks for checking that we had a test to cover None at the top-most level. I suppose it makes sense to test None at the deepest level (which you have) and also somewhere in the middle, to make sure it doesn't mess up subsequent un-nesting operations.

Can you at least add a test with None in one of the middle layers? You can probably simplify the example below (doesn't need to be a direct copy/paste from the issue) and use parametrization to re-use the logic for different inputs (i.e. one with None at the deepest level, one with None somewhere in the middle).  Depending on how that looks maybe you want to even move the example with None at the top level here as well